tidy3d/plugins/mode/solver.py

```diff
@@ -380,11 +383,13 @@ def any_pec(eps_vec, threshold=0.9 * np.abs(pec_val)):
             print(spl.norm(diff, ord="fro"), spl.norm(aca, ord="fro"), spl.norm(aac, ord="fro"))
 
         # Call the eigensolver. The eigenvalues are -(neff + 1j * keff)**2
+
+        print(mat.dtype, eig_guess.dtype, vec_init.dtype)
```

Comment:
This is probably left from debugging, right?

Reply:
yep, removed